packages/gatsby-remark-smartypants/src/index.js

```diff
@@ -1,10 +1,13 @@
 const retext = require(`retext`)
 const visit = require(`unist-util-visit`)
 const smartypants = require(`retext-smartypants`)
+const _ = require('lodash')
 
-module.exports = ({ markdownAST }) => {
+module.exports = ({ markdownAST }, pluginOptions = {}) =>
   visit(markdownAST, `text`, node => {
-    const processedText = String(retext().use(smartypants).process(node.value))
+    const defaultOptions = {}
+    const options = _.defaults(pluginOptions, defaultOptions)
+    const processedText = String(retext().use(smartypants, options).process(node.value))
     node.value = processedText
   })
 }
```

Comment:
this isn't needed anymore. It's good btw to run `npm run format` from the base of the repo on PRs as that'll fix formatting + catch syntax errors like this. Need to create a checklist for PRs :-)